helpers/checks_for_tests.py

```diff
@@ -0,0 +1,48 @@
+"""Typical checks for functional tests."""
+
+from helpers.control import Tempesta
+
+
+def check_tempesta_error_stats(tempesta: Tempesta, cl_msg_parsing_errors: int,
```

Comment:
I can't see `check_tempesta_error_stats` and `check_tempesta_request_and_response_stats`
are used somewhere. But it seems working.

Reply:
These checks are often used in old tests. I use their also (test_purge.py, test_conditional.py)